### ISSUE TITLE
SoftwareSerial doe not compile with ATTiny84

### ISSUE THREAD

**Issue by wholder** [arduino/Arduino#8104]
Select ATTiny84 as the target and then try to compile the SoftwareSerial/SoftwareSerialExample sketch.  On my system using Mac OS 10.13.6 (17G65), Arduino 1.8.5 and DIY Tiny 2018.9.9 I get the compile error show below.  Oddly, if I select ATTiny85, the code compile just fine.

Wayne

/Users/wholder/Library/Arduino15/packages/diy_attiny/hardware/avr/2018.9.9/libraries/SoftwareSerial/SoftwareSerial.cpp: In member function 'void SoftwareSerial::begin(long int)':
/Users/wholder/Library/Arduino15/packages/diy_attiny/hardware/avr/2018.9.9/libraries/SoftwareSerial/SoftwareSerial.cpp:317:37: error: expected primary-expression before ')' token
   if (digitalPinToPCICR(_receivePin)) {
                                     ^
exit status 1
Error compiling for board ATtiny84.

**Comment by per1234** [comment]
This is a bug in the 3rd party diy_attiny hardware package. There is a missing `(` at the start of the `digitalPinToPCICR()` macro at /Users/wholder/Library/Arduino15/packages/diy_attiny/hardware/avr/2018.9.9/variants/tinyX4/pins_arduino.h line 208
```cpp
#define digitalPinToPCICR(p)    ((p) <= 11) ? (&GIMSK) : ((uint8_t *)NULL))
```
I always have trouble navigating that repository so I can't tell whether the issue has been resolved since the 2018.9.9 release. Regardless, this bug has nothing to do with official Arduino code so it's off topic for this issue tracker.

**Comment by wholder** [comment]
Sorry to be off topic, but thanks for the help.

Wayne
